### PR TITLE
Not complete - Missing add movie

### PR DESCRIPTION
**Pull request by lucianosimoni**

### CODE REVIEW

**Comment by lucianosimoni** [comment]
Movie Add not working yet

File: README.md [hunk 1]
```diff
@@ -8,7 +16,7 @@
 
 ## Introduction
 
-You are tasked with building a small frontend application containing 3 forms and a list. There is a screenshot at the bottom of this document that gives you an idea of what to aim for. As you'll be able to see, it doesn't have to *look* good so don't spend time on styling!
+You are tasked with building a small frontend application containing 3 forms and a list. There is a screenshot at the bottom of this document that gives you an idea of what to aim for. As you'll be able to see, it doesn't have to _look_ good so don't spend time on styling!
```
Comment (Joelzor) [inline README.md:19]: is prettier changing this or did you do it manually? .....

File: src/client/App.css [hunk 2]
```diff
@@ -1,42 +1,120 @@
+ul {
```
Comment (Joelzor) [inline src/client/App.css:1]: I like how you wrote lots of CSS instead of finishing the challenge. You are a legend sir

File: src/client/App.css [hunk 3]
```diff
@@ -1,42 +1,120 @@
...
+	padding: 1rem;
+
+	border-radius: 20px;
+	border: 0.2rem solid black;
+	background-color: white;
+
+	cursor: pointer;
+}
+
+@keyframes notification {
```
Comment (Joelzor) [inline src/client/App.css:102]: respect for adding cool animations

File: src/client/App.jsx [hunk 4]
```diff
@@ -1,56 +1,161 @@
...
+		fetch(`${apiUrl}/user/register`, {
+			method: 'POST',
+			headers: { 'Content-Type': 'application/json' },
+			body: JSON.stringify(data),
+		})
+			.then((response) => response.json())
+			.then((data) => {
+				setNotification('');
+				const keyName = Object.keys(data)[0];
+				if (keyName === 'error') setNotification(`${data.error} 🔴`);
```
Comment (Joelzor) [inline src/client/App.jsx:49]: Some nice display messages!

File: src/client/components/LoginStatus.jsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+export default function LoginStatus({ loggedIn, setLoggedIn }) {
+	// useEffect(() => {
+	// 	const user = localStorage.getItem('user');
+	// 	if (user) {
+	// 		setLoggedIn(user);
+	// 	}
+	// }, []);
+
+	function logout() {
```
Comment (Joelzor) [inline src/client/components/LoginStatus.jsx:11]: good extra work with the log in status, I should have done this

File: src/client/components/Notification.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+export default function Notification({ message, setNotification }) {
```
Comment (Joelzor) [inline src/client/components/Notification.jsx:1]: This is cool. I did lots of state values as notifications which is far less effective (wanted to refactor over the weekend but had no time in the end)

File: README.md (hunk 1 above)
Comment (lucianosimoni) [reply README.md:19]: KSDKSAJDKJDKASJDKASDK prettier 😂

**Pull request closed without merging**